src/main/java/ai/pluggy/client/request/ConnectorsSearchRequest.java

```diff
@@ -19,15 +21,16 @@ public ConnectorsSearchRequest(String name, List<String> countries) {
     this(name, countries, Collections.emptyList());
   }
 
-  public ConnectorsSearchRequest(String name, List<String> countries, List<String> types) {
+  public ConnectorsSearchRequest(String name, List<String> countries, List<ConnectorType> types) {
```

Comment:
Also, note that this change will now prevent clients from setting arbitrary types (strings) in this request. This is fine because they will now use the pre-set `ConnectorType` enum but if we ever add new connector types we have to remember to also update this API.